### ISSUE TITLE
Loader breaks on unexpected structure

### ISSUE THREAD

**Issue by rpavlik** [KhronosGroup/OpenXR-SDK-Source#54]
I think we should be able to pass any kind of struct in to any function on the next chain, at least through the loader, but I can't:

```
Error [GENERAL | xrCreateInstance | OpenXR-Loader] : VUID-XrInstanceCreateInfo-next-next: unexpected struct in 'next' chain.
Error [GENERAL | xrCreateInstance | OpenXR-Loader] : VUID-xrCreateInstance-info-parameter: something wrong with XrInstanceCreateInfo contents
Error [GENERAL | xrCreateInstance | OpenXR-Loader] : LoaderInstance::CreateInstance chained CreateInstance call failed
```

This was prompted by this test: https://gitlab.freedesktop.org/monado/utilities/xr-system-tests/blob/master/tests/functions/xrCreateInstance.cpp#L53  which throws an `XrActionCreateInfo` in the `XrInstanceCreateInfo` chain.

**Comment by daveh-lunarg** [comment]
Assigned myself. Buried on another project for a couple weeks, plan to look at this in early July.

**Comment by rpavlik** [comment]
oh go figure - looks mostly like an easy fix - there's a ValidateInstanceCreateInfoNextChain which doesn't really need to exist. Not seeing any associated ones in generated code for the rest - guess I'll have to try messing around with next chains in other functions.

**Comment by daveh-lunarg** [comment]
Closing - resolved by PR #62.
